### DISCUSSION TITLE
Complete disc image converting documentation

### DISCUSSION THREAD

**Discussion by YamamotoKaderate** [claunia/DiscImageChef#714] [Ideas]
I was searching for an open-source tool that support multiple disk image format conversion. 
I'm glad I found Aaru, I like command-line swiss-army knife like ffmpeg. Thank you for your hard work.
But I spend a long time figuring out how to convert some cd disk image. 
I think 2 things are missing in the docs: 
-add examples for disk image with multiple files (CDRWIN CUE+BIN , NERO MDS+MDF)
-put the common file extensions after the name of the file formats in the doc and in Aaru formats

Edit: also the website show an error on firefox if you click on a menu item and then click on an other one.
Edit2: adding support for ecm would be nice. I've seen applyed to cloneCd image, it remove error correction marker to gain size

**Comment by TheRogueArchivist** [comment]
Hello! That's a very good point about not having extensions, as well as not having examples for images with multiple files! I'll make an issue for that right now, and I'll update the documentation accordingly when I have time.

Issue opened: https://github.com/aaru-dps/Aaru.Documentation/issues/15
